genie-client/build.gradle

```diff
@@ -35,38 +35,3 @@ dependencies {
     testCompile("org.springframework.boot:spring-boot-starter-jetty")
     testCompile("org.springframework.cloud:spring-cloud-contract-wiremock")
 }
-
```

Comment:
This doesn't belong with either commits...

Reply:
That's cause it's completely worthless left over dangling code from when @irontable removed the python code from the repository. But sure I can break it into its own commit.